frontend/src/pages/TrainingReportForm/pages/eventSummary.js

```diff
@@ -181,7 +181,7 @@ const EventSummary = ({
                 <Req />
               </Label>
               <Controller
-                render={({ onChange: controllerOnChange, value: id }) => (
+                render={({ onChange: controllerOnChange, value: id, onBlur }) => (
```

Comment:
Not sure if its possible to have event without creator (or to remove it once set).